lib/orbit_web/controllers/react_app_controller.ex

```diff
@@ -6,9 +6,21 @@ defmodule OrbitWeb.ReactAppController do
     # so skip the default app layout.
     render(conn, :react_app,
       layout: false,
-      sentry_dsn: Application.get_env(:sentry, :dsn),
+      appcues_enabled?: Application.get_env(:orbit, :appcues_enabled?),
+      appcues_id: Application.get_env(:orbit, :appcues_id),
+      appcues_uid: appcues_uid(conn),
       environment: Application.get_env(:orbit, :environment),
-      full_story_org_id: Application.get_env(:orbit, :full_story_org_id)
+      full_story_org_id: Application.get_env(:orbit, :full_story_org_id),
+      sentry_dsn: Application.get_env(:sentry, :dsn)
     )
   end
+
+  # Use half a sha256 of the email address as user id.
+  # This is not secure! But we're eliminating randomness
+  #  while not being trivially overridable.
+  defp appcues_uid(conn) do
+    :crypto.hash(:sha3_256, conn.assigns[:email])
```

Comment:
email might be missing (if you haven't logged in I guess?) it's optional in `root.html.heex`. But in that case, we still generate a `uid`. Does that incorrectly imply to fullstory that all logged-out users are the same? Should we avoid generating a uid if there's no email?

Reply:
Orbit redirects logged-out users to SSO before they can get to `react_app_controller`, so the optional in `root.html.heex` is purely caution (unnecessary?). Both appcues and fullstory will only ever be seeing logged-ins.